_posts/2022-12-31-end-of-year-update.markdown

```diff
@@ -49,26 +49,26 @@ When Qt6 came around, they switched to CMake, which is a lot nicer for cross-com
 
 ![gtk3-demo](/assets/2022-end-of-year-update/gtk3-demo.png)
 
 ## Other projects updates
 
 ### mlibc on Linux ([repo](https://github.com/no92/linux-mlibc))
-While working towards running a proper browser on Managarm, we came to the realization that we can't rule out whether the kernel / servers are wrong (A.K.A. Managarm itself), or if mlibc had a bug. To eliminate the first possibility and granting easy access to lots of nice debug tools, we started the [mlibc LFS](https://github.com/no92/linux-mlibc) project, named after Linux From Scratch. It's essentially Linux From Scratch, but using mlibc instead of glibc.
+While working towards running a proper browser on Managarm, we came to the realization that we can not rule out whether the kernel / servers are wrong (A.K.A. Managarm itself), or if mlibc had a bug. To eliminate the first possibility and granting easy access to lots of nice debug tools, we started the [mlibc LFS](https://github.com/no92/linux-mlibc) project, named after Linux From Scratch. It is essentially Linux From Scratch, but using mlibc instead of glibc.
 
 We quickly got a small userspace running in a chroot, but it eventually grew out to something that we can boot up in a VM, a neccesity when testing out graphical programs. The major things that came from this still ongoing project is the fixes and additions to the mlibc Linux ABI, general usability under Linux, and an easier way to debug applications in the future. While we did get to a point where we could run WebKitGTK reasonably, this has since regressed on a nasty deadlock in pthreads, which we're working on fixing.
 
 ## What do we want to achieve in the next year?
 
-**Finish porting xbps.** Unfortunately, we didn't get exactly where we wanted to be this year on this point, even though the netstack got some nice updates and is a lot more stable now. We aim to have `xbps` in a state where an end user can just update the package repository and install packages by the end of next year, on supported network hardware.
+**Finish porting xbps.** Unfortunately, we did not get exactly where we wanted to be this year on this point, even though the netstack is a lot more stable now. We aim to have `xbps` in a state where an end user can just update the package repository and install packages by the end of next year, on supported network hardware.
```

Comment:
First sentence reads a bit clunky, could just be 'Unfortunately, despite improvements to the network stack, we did not finish porting `xbps` this year'.

Reply:
Applied, thanks!